util/util.go

```diff
@@ -0,0 +1,61 @@
+package util
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/bmatcuk/doublestar"
+)
+
+func CreateFolderInCurrentDirectory(dirname string) error {
+	currentDir, _ := os.Getwd()
+	p := filepath.Join(currentDir, dirname)
+	err := os.Mkdir(p, 0700)
+	if os.IsExist(err) {
+		fmt.Printf("skipping creation of folder '%s' because it already exists \n", dirname)
+	} else if err != nil {
+		fmt.Println(err)
+		return err
+	}
+	return nil
+}
+
+func SafeWriteFileOrExit(filename string, fileContent []byte) error {
+	if _, err := os.Stat(filename); os.IsNotExist(err) {
+		err := ioutil.WriteFile(filename, fileContent, 0777)
+		if err != nil {
+			return err
+		}
+	} else {
+		fmt.Printf("skipping creation of file %s because it already exists\n", filename)
+		if filename == ".gitignore" {
+			fmt.Println(`add these enties on your .gitignore manually
+
+*.retry
+.terraform
+*.tfstate
+*.backup
+			`)
+		}
+		os.Exit(0)
+	}
+	return nil
+}
+
+func FindBasesFromVendor(vendorPath string) (paths []string, err error) {
```

Comment:
I would move this function to a more package with a more "speaking" name, such as `furyfile`